python-package/xgboost/dask.py

```diff
@@ -1544,15 +1543,21 @@ def inplace_predict(  # pylint: disable=unused-argument
 
 
 async def _async_wrap_evaluation_matrices(
-    client: Optional["distributed.Client"], **kwargs: Any
+    client: Optional["distributed.Client"],
+    tree_method: Optional[str],
+    max_bin: Optional[int],
+    **kwargs: Any,
```

Comment:
It appears that `tree_method` and `max_bin` will no longer be in the `kwargs` dictionary. Can you make sure this will not cause undesirable behavior? For example, is `kwargs` passed to `xgb.train`?

Reply:
Training parameters are obtained via `params = self.get_xgb_params()`, which you can find in the `async def _fit_async` function.